src/CalendarInput/index.js

```diff
@@ -45,6 +45,13 @@ const isPeriodSelection = equals(PERIOD_SELECTION)
  */
 class CalendarInput extends Component {
   constructor (props) {
+    if (process.env.NODE_ENV !== 'test') {
+      // eslint-disable-next-line no-console
+      console.warn(
+        'CalendarInput component is being deprecated, use DateInput with showCaledar prop as "false" instead.'
+      )
+    }
+
     super(props)
     const { start, end } = props.value
     const isVaidStart = (!start && !isPeriodSelection(props.dateSelection))
```

Comment:
typooooo!!1!!! but I recommend fixing this in line 60 too.
```suggestion
    const isValidStart = (!start && !isPeriodSelection(props.dateSelection))
```